ParallaxGenLib/src/BethesdaDirectory.cpp

```diff
@@ -629,6 +635,18 @@ void BethesdaDirectory::updateFileMap(const filesystem::path &FilePath,
   FileMap[LowerPath] = NewBFile;
 }
 
+auto BethesdaDirectory::isFileInBSA(const filesystem::path &File, const std::set<std::wstring> &BSAFiles) -> bool {
+  if (isBSAFile(File)) {
+    BethesdaFile const BethFile = getFileFromMap(File);
+    std::filesystem::path const BsaFilename = BethFile.BSAFile->Path.filename();
+
+    if (BSAFiles.find(BsaFilename.wstring()) != BSAFiles.end()) {
```

Comment:
Probably in ParallaxGenConfig we should lowercase all the BSAfiles entries before adding them to the set so that this comparison is case insensitive

Reply:
Wouldn't that hurt performace sincd the BsaFilename would need to be converted to lower case here as well? What would be the benefit?

Reply:
Maybe we should convert BSAFile->Path to lowercase in `BethesdaDirectory::addBSAToFileMap`? This comparison does need to be case insensitive because NTFS is case insensitive (I wish it wasn't). If we convert to lowercase in the addBSAToFileMap method and in ParallaxGen config then no conversion will be necessary in the function you wrote. What do you think?

Reply:
I made the lookup case insensitive with a predicate, had no effect on the time parallaxgen took.